.tekton/integration/integration-test.yaml

```diff
@@ -0,0 +1,199 @@
+        - image: registry.redhat.io/openshift4/ose-cli:latest
+          resources:
+            requests:
+              memory: 4Gi
+            limits:
+              memory: 4Gi
+          env:
+          - name: SNAPSHOT
+            value: $(params.SNAPSHOT)
+          - name: KUBECONFIG
+            value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+          volumeMounts:
+            - name: credentials
+              mountPath: /credentials
+          script: |
+            dnf -y install jq git golang make
+            echo ${SNAPSHOT} | jq .
+            env
+
+            OPERATOR_IMAGE=$(echo ${SNAPSHOT} | jq -r '.components[] | select(.name=="kueue-operator-main") | .containerImage')
+            OPERAND_IMAGE=$(echo ${SNAPSHOT} | jq -r '.components[] | select(.name=="kueue-operand-main") | .containerImage')
+            MUSTGATHER_IMAGE=$(echo ${SNAPSHOT} | jq -r '.components[] | select(.name=="kueue-must-gather-image") | .containerImage')
+            SRC_REPO=$(echo ${SNAPSHOT} | jq -r '.components[] | select(.name=="kueue-operator-main") | .source.git.url')
+            SRC_REVISION=$(echo ${SNAPSHOT} | jq -r '.components[] | select(.name=="kueue-operator-main") | .source.git.revision')
+            SRC_BASEDIR=$(basename -s .git $SRC_REPO)
+            NAMESPACE=openshift-kueue-operator
+
+            OPERATOR_IMAGE_ORIGINAL=registry.redhat.io/kueue/kueue-rhel9-operator:latest
+            OPERAND_IMAGE_ORIGINAL=registry.redhat.io/kueue/kueue-rhel9:latest
+            MUSTGATHER_IMAGE_ORIGINAL=registry.redhat.io/kueue/kueue-must-gather-rhel9:latest
+
+            echo "----------------------------"
+            echo "Images to be tested:"
+            echo "Kueue operator: ${OPERATOR_IMAGE}"
+            echo "Kueue operand:  ${OPERAND_IMAGE}"
+            echo "Kueue must-gather:  ${MUSTGATHER_IMAGE}"
+            echo "----------------------------"
+
+            git clone $SRC_REPO
+            pushd $SRC_BASEDIR
+            git checkout $SRC_REVISION
+            make deploy-upstream-cert-manager
+
+            TMPDIR=$(mktemp -d)
+            cp -r deploy ${TMPDIR}
+            pushd $TMPDIR
+
+            sed -i "s|${OPERATOR_IMAGE_ORIGINAL}.*|${OPERATOR_IMAGE}|g" deploy/07_deployment.yaml
+            sed -i "s|${OPERAND_IMAGE_ORIGINAL}.*|${OPERAND_IMAGE}|g" deploy/07_deployment.yaml
+            if [ -n "${MUSTGATHER_IMAGE}" ]; then
+              sed -i "s|${MUSTGATHER_IMAGE_ORIGINAL}.*|${MUSTGATHER_IMAGE}|g" deploy/07_deployment.yaml
+            fi
+
+            oc create ns ${NAMESPACE}
+            oc apply -f deploy/
+            oc apply -f deploy/crd
+            popd
+
+            # wait for operator
+            timeout 300s bash -c 'until oc get deployment openshift-kueue-operator -n openshift-kueue-operator -o jsonpath="{.status.conditions[?(@.type==\"Available\")].status}" | grep -q "True"; do sleep 10; echo "Still waiting..."; done'
+
+            if make e2e-ci-test; then
```

Comment:
This only runs our operator e2e.

We also have https://github.com/openshift/kueue-operator/blob/main/Makefile#L214.